### PR TITLE
Added a new section on Azure CNI

### PR DESCRIPTION
**Pull request by aanandr**
Azure provides a CNI for native integration of Kubernetes Pods in a Virtual Network. I have added a short description of this CNI and provided links to important documentation

### CODE REVIEW

**Comment by aanandr** [comment]
Fixed the check that failed

**Comment by aanandr** [comment]
#lgtm #approved

**Comment by aanandr** [comment]
cncf-cla:yes language/en sig/docs size/XS

**Comment by Rajakavitha1** [comment]
Hi @aanandr !!! Could you please incorporate the changes that @sftim has suggested.

**Comment by aanandr** [comment]
Thanks for reviewing this. I have made some changes per your recommendation

**Comment by Rajakavitha1** [comment]
@sftim please take a look at the updated changes and state your approval.

**Comment by neolit123** [comment]
this document is technically owned by sig-network.

/sig network

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -106,6 +106,14 @@ Using this CNI plugin allows Kubernetes pods to have the same IP address inside
 
 Additionally, the CNI can be run alongside [Calico for network policy enforcement](https://docs.aws.amazon.com/eks/latest/userguide/calico.html). The AWS VPC CNI project is open source with [documentation on GitHub](https://github.com/aws/amazon-vpc-cni-k8s).
 
+### Azure CNI for Kubernetes
+
+[Azure CNI](https://docs.microsoft.com/en-us/azure/virtual-network/container-networking-overview) is an [open source](https://github.com/Azure/azure-container-networking/blob/master/docs/cni.md) plugin that brings the rich set of Azure Networking capabilities to Pods running in a Kubernetes cluster by utilizing the same SDN stack that powers Virtual Machines. It assigns VNet IP addresses to Pods and connects them directly to other Pods and VNet resources, providing performance that is at par with VMs. Pods can also connect seamlessly to peered VNets and to on-premises over Express Routes or site-to-site VPN and are also directly reachable from these networks. Pods can access Azure services such as storage and SQL that are protected by Service Endpoints or Private Link. VNet routing and security policies can be used for filtering Pods traffic and securing Kubernetes clusters.
```
[the remark this answers is not in the document]
Comment (aanandr) [reply content/en/docs/concepts/cluster-administration/networking.md]: Thanks for these suggestions. I updated the text again and resubmitted. The new text is closer to the second suggestion you provided. I just added an extra line.

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -106,6 +106,14 @@ Using this CNI plugin allows Kubernetes pods to have the same IP address inside
...
+
+Azure CNI sets up networking for Pods by utilizing a pool of secondary Vnet IPs pre-configured on the Network Interface of a Kubernetes node. Pods that come up in a node are assigned IPs from the pool configured on the node and when the pod terminates its IP is returned back to the pool.
```
[the remark this answers is not in the document]
Comment (aanandr) [reply content/en/docs/concepts/cluster-administration/networking.md]: This makes sense. Who can make this change?

[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/concepts/cluster-administration/networking.md]: @aanandr feel free to log an issue, and mention me in a comment - I'll spot it.
It'd be a low-priority, nice-to-have bit of cleanup.

**Comment by aanandr** [comment]
@sftim  - thanks Tim. Do you have to put lgtm labels to formally approve this?

**Comment by sftim** [comment]
LGTM is largely for technical signoff. I've no reason to doubt that Azure CNI works as described, but I can't say that I've checked.

**Comment by aanandr** [comment]
@sftim  thanks! The review status says that the PR is not mergeable and needs approved lgtm labels. Hence i asked. How do i make this mergeable?

**Comment by aanandr** [comment]
I just saw that one more review is pending. @thockin can you please review this?

**Comment by aanandr** [comment]
@thockin can you please review this PR? Its been pending for some time now. @sftim has already reviewed and signed off.

**Comment by aanandr** [comment]
@Rajakavitha1 - can you help me get this PR approved and merged? Its been open for a long time now. Let me know if there is any action that i need to take.

**Comment by sftim** [comment]
Would anyone from @kubernetes/sig-network-pr-reviews be willing to take a look and comment on technical accuracy?

**Comment by chandanAggarwal** [comment]
approved

**Comment by aanandr** [comment]
It looks like "approved" label is required from @feiskyer  or @saiyan86 . Can you please do that?

**Comment by feiskyer** [comment]
The changes look good. @sftim could you help to approve?

/assign @sftim

**Comment by sftim** [comment]
Usually, the [PR wrangler](https://github.com/kubernetes/website/wiki/PR-Wranglers) of the week will catch up on /lgtm and other reviews and mark approval if everything looks good.

This one does!
/approve

**Comment by aanandr** [comment]
Thanks a lot @sftim ! :)

**Pull request merged**